pkg/3scale/amp/operator/redis_reconciler.go

```diff
@@ -124,3 +147,100 @@ func Redis(apimanager *appsv1alpha1.APIManager, client client.Client) (*componen
 	}
 	return component.NewRedis(opts), nil
 }
+
+func (r *RedisReconciler) deleteSystemRedisSecretNamespaceAttribute() (reconcile.Result, error) {
```

Comment:
No point in returning `reconcile.Result` if the only possible result is {}

Reply:
Done. Thank you